src/game/client/shadow/w3dbuffermanager.cpp

```diff
@@ -0,0 +1,475 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+
+#include "w3dbuffermanager.h"
+#include "dx8indexbuffer.h"
+#include "dx8vertexbuffer.h"
+
+// TODO need general cleanup advice
+// TODO what is going on with count
+
+static int FVFTypeIndexList[W3DBufferManager::MAX_FVF] = {
+    D3DFVF_XYZ | D3DFVF_TEX0, // XYZ
```

Comment:
Should we use the enum's for this? https://github.com/TheAssemblyArmada/Thyme/blob/43307542d7b72507a5dbdfa275364fe252452a8f/src/w3d/renderer/dx8fvf.h#L21-L33